### ISSUE TITLE
Failed to send the command after 3 attempts (Status Fail) (ZW0202)

### ISSUE THREAD

**Issue by Snapperud** [hassio-addons/addon-zwavejs2mqtt#317]
Hi
I have a problem for a few months, running Z-Wave JS to MQTT 0.32.1 and HA 2021.12.3.
Zwave Controller Aeotec Z-Stick 7 FW 7.15.
Have tested with extension cable and both USB-2 and USB-3.
The only solution is to restart Z-Wave JS to MQTT Add-on, then everything is ok, to the next crash.
it can happen 2-4 a day , here is the Zwave log:

2021-12-16 00:58:37.425 INFO ZWAVE: Node 12: value updated: 49-4-Air temperature 22.8 => 22.8
2021-12-16 00:58:41.480 INFO ZWAVE: Node 18: value updated: 67-0-setpoint-1 18 => 18
2021-12-16 00:58:45.573 INFO ZWAVE: Node 13: value updated: 67-1-setpoint-1 22 => 25.5
2021-12-16 00:58:47.817 INFO ZWAVE: Node 13 is now dead
2021-12-16 00:58:47.926 ERROR ZWAVE-SERVER: Failed to send the command after 3 attempts (Status Fail) (ZW0202)
ZWaveError: Failed to send the command after 3 attempts (Status Fail) (ZW0202)
at Driver.sendMessage (/opt/node_modules/zwave-js/src/lib/driver/Driver.ts:3345:23)
at Driver.sendCommand (/opt/node_modules/zwave-js/src/lib/driver/Driver.ts:3504:28)
at ThermostatSetpointCCAPI.set (/opt/node_modules/zwave-js/src/lib/commandclass/ThermostatSetpointCC.ts:260:21)
at Proxy.ThermostatSetpointCCAPI.<computed> (/opt/node_modules/zwave-js/src/lib/commandclass/ThermostatSetpointCC.ts:184:14)
at ZWaveNode.setValue (/opt/node_modules/zwave-js/src/lib/node/Node.ts:849:14)
at Function.handle (/opt/node_modules/@zwave-js/server/dist/lib/node/message_handler.js:20:44)
at Object.node (/opt/node_modules/@zwave-js/server/dist/lib/server.js:39:91)
at Client.receiveMessage (/opt/node_modules/@zwave-js/server/dist/lib/server.js:95:99)
at WebSocket.<anonymous> (/opt/node_modules/@zwave-js/server/dist/lib/server.js:48:45)
at WebSocket.emit (node:events:390:28)
2021-12-16 00:58:51.160 INFO ZWAVE: Node 13: has returned from the dead

2021-12-16 12:20:12.409 INFO ZWAVE: Node 7: value updated: 37-0-currentValue false => true
2021-12-16 12:20:12.410 INFO ZWAVE: Node 7: value updated: 37-0-targetValue false => true
2021-12-16 12:20:12.524 INFO ZWAVE: Node 7: value updated: 37-0-currentValue true => true
2021-12-16 12:20:13.892 INFO ZWAVE: Node 7: metadata updated: 49-0-Power
2021-12-16 12:20:13.893 INFO ZWAVE: Node 7: value updated: 49-0-Power 0 => 2.2
2021-12-16 12:20:14.241 INFO ZWAVE: Node 7: metadata updated: 49-0-Power
2021-12-16 12:20:14.242 INFO ZWAVE: Node 7: value updated: 49-0-Power 2.2 => 2.2
2021-12-16 12:20:14.348 INFO ZWAVE: Node 7: metadata updated: 49-0-Power
2021-12-16 12:20:14.348 INFO ZWAVE: Node 7: value updated: 49-0-Power 2.2 => 2.2
2021-12-16 12:20:15.475 INFO ZWAVE: Node 7 is now dead
2021-12-16 12:20:15.585 ERROR ZWAVE-SERVER: Failed to send the command after 3 attempts (Status Fail) (ZW0202)
ZWaveError: Failed to send the command after 3 attempts (Status Fail) (ZW0202)
at Driver.sendMessage (/opt/node_modules/zwave-js/src/lib/driver/Driver.ts:3345:23)
at Driver.sendCommand (/opt/node_modules/zwave-js/src/lib/driver/Driver.ts:3504:28)
at BinarySwitchCCAPI.set (/opt/node_modules/zwave-js/src/lib/commandclass/BinarySwitchCC.ts:114:21)
at Proxy.BinarySwitchCCAPI.<computed> (/opt/node_modules/zwave-js/src/lib/commandclass/BinarySwitchCC.ts:129:14)
at ZWaveNode.setValue (/opt/node_modules/zwave-js/src/lib/node/Node.ts:849:14)
at Function.handle (/opt/node_modules/@zwave-js/server/dist/lib/node/message_handler.js:20:44)
at Object.node (/opt/node_modules/@zwave-js/server/dist/lib/server.js:39:91)
at Client.receiveMessage (/opt/node_modules/@zwave-js/server/dist/lib/server.js:95:99)
at WebSocket.<anonymous> (/opt/node_modules/@zwave-js/server/dist/lib/server.js:48:45)
at WebSocket.emit (node:events:390:28)
2021-12-16 12:20:15.625 INFO ZWAVE: Node 7: metadata updated: 49-0-Power
2021-12-16 12:20:15.626 INFO ZWAVE: Node 7: value updated: 49-0-Power 2.2 => 2.2
2021-12-16 12:20:15.627 INFO ZWAVE: Node 7: has returned from the dead

**Comment by froguinou** [comment]
I have the same problem

**Comment by Snapperud** [comment]
There are different nodes that fails, from time to time. 
Today i installed an Aeotec Range Extender 7 and run a "Heal network", but it did not get any better. :-(

**Comment by Snapperud** [comment]
Have created a case here, because I think the problem is on zwavejs / zwavejs2mqtt and not in HA.
https://github.com/zwave-js/zwavejs2mqtt/issues/2082

**Comment by MRobi1** [comment]
Seeing this exact same issue on many devices. Most notably after a restart or when controlling multiple devices at the same time. For example, I have an automation when double tapping on a switch it will turn on 4 lights, 2 plugs and open a garage door. About 50% of the time when running this automation, I'll get this error on at least 1 of those devices which causes it to show as unavailable. I can either re-interview the node or restart the zwavejs2mqtt addon and all is well again.

**Comment by Wouter0100** [comment]
I had the same issue a couple of times already. Restarting the addon solves it. 

While in this state, no actions succeed.

**Comment by johanschelin** [comment]
This is a known bug with the 700-chip sticks. I have moved from Aeotec Z-Stick 7 to the Gen5+ which solves the issue until SiLabs succeeds with a new firmware fix for the 700-chips. 
Please read and follow this thread for information https://github.com/zwave-js/node-zwave-js/issues/3906

**Comment by Wouter0100** [comment]
@johanschelin thanks for mentioning this. I researched it a bit, but I'm curious as I have never had this issue in the past and I have a Z-Stick Gen5 all along. 

As far as I know, the NVM (Non-Volatile Memory?) shouldn't update itself from a 500 to 700, is it?

**Comment by johanschelin** [comment]
Hi Wouter0100 - ok, if you are experiencing this with a Gen5 stick, it is something else - but the 7-sticks all seem to have this issue :)

The NVM cant update, as 500/700 are different hardware in the sticks

**Comment by MRobi1** [comment]
I updated the firmware in my zooz zst 700 and this issue has been significantly better. I've had 1 entity marked as unavailable over the past 2 weeks. Huge improvement over having 5+ unavailable daily.

**Comment by Wouter0100** [comment]
I'm on the latest version and still having this issue from time to time.

**Comment by jamespreedy** [comment]
Similar - I'm on the latest version and have this too. I'm on a Z-Stick 500 series, not 700 series.
Thanks in advance.

**Comment by kdknigga** [comment]
All, you should probably report this to the actual zwavejs2mqtt project.

https://github.com/zwave-js/zwavejs2mqtt

This repo is just the glue that brings zwavejs2mqtt into Home Assistant.  No actual work is done here.

**Comment by Wouter0100** [comment]
Thanks @kdknigga.

If anyone is having this issue again, would you be able to create a dump using:
Home Assistant -> settings -> Integrations -> Z-Wave JS -> Configure -> Create dump -> zip the json file and post it here.

This is as asked by the issue form of the zwavejs2mqtt project. I will hopefully be able to reproduce it myself, as I had this issue again yesterday. After this dump is available, we should be able to move this issue to the zwavejs2mqtt project.

**Comment by Lonestar10** [comment]
I was in the process of adding a fifth Kwikset Zwave lock to my new x86-64 HA OS system when HA started having trouble.  When I asked HA to re-interview the lock, the entire 30 device Zwave network went haywire and hasn't come back.  I'm using a Zooz ZST10 700 (firmware 7.17.2) on a six-foot extension. I've moved it to various locations, but I still can't control any Zwave device.
I've unplugged the Zooz dongle, powered off the x86-64 and restarted numerous times, but no luck.
I've used VeraPlus for almost 10 years and never had this kind of problem (it didn't make you jump through hoops--Keymaster-- to add a lock, either).
I'm using ZwaveJS2MQTT..at least I THINK I am.  I also see ZwaveJS on my integrations page.  The whole Zwave integration is very confusing.
Home Assistant -> settings -> Integrations -> Z-Wave JS -> Configure -> Create dump -> zip the json file.  I don't see a Create Dump file.
